src/WebJobs.Extensions.DurableTask/Scale/ScaleUtils.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#nullable enable
+
+using System;
+using System.Threading.Tasks;
+
+#if !FUNCTIONS_V1
+using Microsoft.Azure.WebJobs.Host.Scale;
+#endif
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Scale
+{
+    internal static class ScaleUtils
+    {
+#if !FUNCTIONS_V1
+        internal static IScaleMonitor GetScaleMonitor(DurabilityProvider durabilityProvider, string functionId, FunctionName functionName, string? connectionName, string hubName)
```

Comment:
Can we also add unit tests for `ScaleUtils.GetTargetScaler` and  `ScaleUtils.GetScaleMonitor`

Reply:
Just added them: https://github.com/Azure/azure-functions-durable-extension/pull/2462/commits/635aa753269dc859ceb27242fd45856117ce9582

Reply:
and expanded them here: https://github.com/Azure/azure-functions-durable-extension/pull/2462/commits/3ad0f2897415867cfa9faaaea8cbeb505c3d0b92